spec/features/lender_contributes_to_loan_request_spec.rb

```diff
@@ -7,13 +7,13 @@
                              role: 0)
   }
 
-  let!(:borrower) { User.create(email: "jeff@example.com",
+  let(:borrower) { User.create(email: "jeff@example.com",
                                 password: "password",
                                 name: "jeff",
                                 role: 1)
   }
 
-  let!(:loan_request) { LoanRequest.create(title: "Farm Tools",
+  let(:loan_request) { LoanRequest.create(title: "Farm Tools",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.